org.eclipse.winery.topologymodeler.ui/src/app/node/node.component.ts

```diff
@@ -62,15 +50,11 @@ export class NodeComponent implements OnInit, AfterViewInit, DoCheck, OnDestroy
   @Output() addNodeToDragSelection: EventEmitter<any>;
   @Output() checkIfNodeInSelection: EventEmitter<string>;
   @Input() selectedNodes: any[] = [];
```

Comment:
Please check if you really need `any` here. I think you know which type this variable should have.

Also: Please check this at **every** `any`. If no type can be applied, just give a short comment why you're using `any`.